libipl/p10/common.H

```diff
@@ -89,4 +89,14 @@ int ipl_set_sbe_state_all(enum sbe_state state);
  */
 int ipl_set_sbe_state_all_sec(enum sbe_state state);
 
+/*
+ * @Brief Process PLAT error to add callout/deconfig and callback
```

Comment:
- @brief?
- Looks, you are using this for the clock error so can you add appropriate documentation?

Reply:
ok